src/front-end/src/identification/Users/components/UserListRow.tsx

```diff
@@ -7,12 +7,15 @@ import {
   RoleActions,
   RoleRequest,
   Roles,
-  User,
+  // User,
 } from "../../../api/types/friendly";
 import { modifyAccessToken, useGlobalState } from "../../../GlobalState";
 
+import { MockUser } from "../../../mock-data-util/mock-interface";
+
 interface UserListRowProps {
-  user: User;
+  // ** User
+  user: MockUser;
```

Comment:
I think if I run CI these changes will break tests.

Reply:
I will definitely fix this first.